### ISSUE TITLE
`#lang errortrace racket` fails

### ISSUE THREAD

**Issue by samth** [racket/errortrace#3]
It produces this error:

```
[samth@huor:~/sw/plt (travis-pkg-test) plt] r /tmp/x.rkt
errortrace/errortrace-lib/errortrace/lang/reader.rkt:16:48: module: initial import is not a well-formed module path
  at: (#%module-begin racket (#%module-begin))
  in: (module n (#%module-begin racket (#%module-begin)))
  context...:
   /home/samth/sw/plt/errortrace/errortrace-lib/errortrace/lang/body.rkt:11:0
```

**Comment by florence** [comment]
@mflatt I think this is an expander bug?

The reader for `#lang errortrace` will rewrite (module n l . body) to be (module n errortrace/lang/body (#%module-begin l . body)).

But when #%module-begin expands the form its expanding is (#%module-begin (#%module-begin l . body)). It seems like the expander is inserting an extra #%module-begin for some reason?


Concretely:

```
#lang errortrace racket
```

Reads as `(module name errortrace/lang/body (#%module-begin racket (#%module-begin)))`

but the syntax passed to module-begin is: `(#%module-begin (#%module-begin racket (#%module-begin)))`

**Comment by florence** [comment]
This seems to have been fixed by b3f5868eed64bc8ab59439aa21f20280f599fe77
